src/Zenaton/v1/Engine/Engine.php

```diff
@@ -31,7 +34,7 @@ public function setProcessor($processor)
     public function execute($jobs)
     {
         // check arguments'type
-        $this->checkArguments($jobs);
+        $this->checkArguments($jobs, __METHOD__);
```

Comment:
I imagine `__METHOD_` is an built-in alias for the method name where this is called from? In this case `execute`?

Reply:
Exactly.